CONTRIBUTING.md

```diff
@@ -72,6 +72,24 @@ plus various other directories with `regression-tests.*` names.
 * The rest of the commit body should be wrapped at 72 characters (see [this](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info)
 * If this commit fixes an issue, put "Closes #XXXX" in the message
 * Do not put whitespace fixes/cleanup and functionality changes in the same commit
+* Include a valid "Signed-off-by" line as a [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin), version [1.1](https://github.com/PowerDNS/pdns/blob/master/DCO)
+
+# Developer Certificate of Origin
+
+We require a "Signed-off-by" on all commits contributed to the PowerDNS codebase, as a [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin), version [1.1](https://github.com/PowerDNS/pdns/blob/master/DCO)
```

Comment:
This should really have a "as of _some release_" statement, as it is not a statement about the entirety of commits to the repository.

Reply:
I'm not sure mentioning a release here makes sense:
- we will start requiring SOB for contributions to any branches as soon as this PR is merged
- people reading this document are willing to contribute new code, and usually don't care about what we did in the past. The document lists a lot of steps (code formatting, spelling, static analyzers, ...) that are now required but weren't in the past.